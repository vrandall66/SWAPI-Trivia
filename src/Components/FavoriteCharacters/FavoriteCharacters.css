.FavoriteCharacters {
  display: flex;
  flex-direction: column;
}


.toMoviesBtn__div {
  display: flex;
}

.backToMovies{
  margin: 0;
  font-family: "Roboto Condensed", sans-serif;
  font-size: 30px;
  color: #ffe81f;
  text-shadow: 2px 2px 2px#222;
  cursor: pointer; 
  height: 42px;
}

.noFavText {
  color: #ffe81f;
  font-size: 50px;
}

.back--episodes--saber {
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: 10px;
}
.backToMovies {
  border: none;
  border-radius: 0 50px 50px 0;
  height: 30px;
  background: #0000e1;
  font-size: 20px;;
  box-shadow: 10px 7px 34px -3px #0000e1;
  cursor: pointer; 
}
