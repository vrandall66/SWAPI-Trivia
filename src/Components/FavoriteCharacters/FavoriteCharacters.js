import React from 'react';
import Character from '../Character/Character'
import '../Character/Character.css'
import './FavoriteCharacters.css'

const FavoriteCharacters = ({ characterInfo, updateCharacterFavorite}) => {


  return (
     <section className="CharactersContainer-section">
      <Character characterInfo={characterInfo} updateCharacterFavorite={updateCharacterFavorite} />
     </section>
  )
}

export default FavoriteCharacters;