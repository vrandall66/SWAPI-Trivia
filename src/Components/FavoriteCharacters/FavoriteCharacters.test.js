import React from "react";
import { shallow } from "enzyme";
import FavoriteCharacters from "./FavoriteCharacters";

describe("FavoriteCharacters", () => {
  let wrapper;
  let favorites = [];
  let addFavoriteCharacter = jest.fn();

  beforeEach(() => {
    wrapper = shallow(
      <FavoriteCharacters
        characterInfo={favorites}
        addFavoriteCharacter={addFavoriteCharacter}
      />
    );
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
