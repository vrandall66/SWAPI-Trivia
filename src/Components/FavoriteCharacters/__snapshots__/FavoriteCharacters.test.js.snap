// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoriteCharacters should match the snapshot 1`] = `
<div
  className="FavoriteCharacters"
>
  <div
    className="toMoviesBtn__div"
  >
    <div
      className="back--episodes--saber"
    >
      <Link
        to="/movies"
      >
        <div
          className="saberButt sbleft"
        />
      </Link>
      <Link
        to="/movies"
      >
        <button
          className="backToMovies"
          type="button"
        >
          Back To All Episodes
        </button>
      </Link>
    </div>
  </div>
  <h3
    className="noFavText"
  >
    Favorite Characters:
  </h3>
  <section
    className="CharactersContainer-section"
  >
    <h3
      className="noFavText"
    >
      No Character's Favorited
    </h3>
  </section>
</div>
`;
